documentation/Experian Commerce Cloud Integration Manual.md

```diff
@@ -281,6 +282,11 @@ To check the imported site preferences go to **Administration > Site Development
 
 ### 3.3\. Testing
 For both (Sitegenesis and Storefront Reference Architecture) stores the testing touchpoints are the same.
+Automated tested are located in tests\tests\
+To run the tests:
+* Install intern (npm i intern).
+* If needed you'll have to install npx (npm install -g npx)
+* Run the command npx intern
```

Comment:
This is correct, but technically just can just install all of the dependencies using `npm install`. Also, for commands, use the backticks  (usually to the left of 1) so it looks like `this`.